### PR TITLE
Added y_lb_demo content to requitements

### PR DESCRIPTION
**Pull request by anpolimus**
Added y_lb_demo_content to the WS profile requirements.

Screenshot 2023-03-02 at 17 17 10

## Steps for review

- [ ] Check that install was successful
- [ ] Go to admin/modules and check that Y Layout Builder Demo Content is available.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -140,6 +140,7 @@
         "open-y-subprojects/openy_content_core": "^1.1",
         "open-y-subprojects/common_scss": "dev-main",
         "ycloudyusa/y_lb": "^1",
+         "ycloudyusa/y_lb_demo_content": "^1",
```
Comment (podarok) [inline composer.json:143]: there is no version yet, let's do like below
```suggestion
        "ycloudyusa/y_lb_demo_content": "^1 || dev-main",
```

**Comment by podarok** [comment]
Go for it @anpolimus 
We need this for development
For the distribution we can discuss this later

**Comment by anpolimus** [comment]
retest this please

**Comment by podarok** [comment]
We need to decide how we add demo content for Layout Builder epic
We have Standard/Extended/Custom and given we are moving from paragraphs to lb, we should add it to Standard profile, enabled by default in order for QA process to become smooth and for end users to avoid any manual steps for enabling lb

**Comment by podarok** [comment]
retest this please

**Pull request merged**